articles/sso/v2/single-page-apps-sso.md

```diff
@@ -0,0 +1,177 @@
+---
+description: How to implement client-side SSO on single page applications.
+toc: true
+---
+
+# Client-Side SSO on Single Page Applications
+
+Single Page Applications (SPA) are user-friendly apps that load a single HTML page. This page then dynamically updates as the users interacts with the app. If your SPA is associated with other apps or sites that asks for authentication, you can implement OIDC-compliant Single Sign On to minimize the number of times the user has to provide their credentials. This ensures both the security of the process and ease of use from the perspective of the user.
+
+## What SSO Looks Like
+
+Suppose you have three applications:
+
```

Comment:
This is the "legacy" scenario. I think in this p2 case the sample app has only 1 SPA, right?